pureport/exception/api.py

```diff
@@ -8,6 +8,16 @@ class MissingAccessTokenException(IOError):
     pass
 
 
+class ConnectionFailedToBecomeActiveException(IOError):
```

Comment:
`OperationTimeoutException` might be a better name for this.  I could also see this used when waiting for a connection to be updated or deleted.